### PR TITLE
Pm 2020 03 comments style clean

### PR DESCRIPTION
**Pull request by phillimorland**
a4 dependency will need updating

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,8 +20,8 @@
     "@babel/runtime": "7.9.2",
     "@fortawesome/fontawesome-free": "5.13.0",
     "acorn": "7.1.1",
-    "adhocracy4": "liqd/adhocracy4#aplus-v2002.4",
-    "autoprefixer": "9.7.5",
+    "adhocracy4": "liqd/adhocracy4#5f83e0a2970c79476206d4d0830e1a3cc265e73a",
+    "autoprefixer": "9.7.4",
```
Comment (fuzzylogic2000) [inline package.json:24]: Is this on purpose or ended up in here by mistake?

Comment (phillimorland) [reply package.json:24]: the auto prefixer no, but the hash I left as need to update it anyway whic I will do now

**Comment by phillimorland** [comment]
@fuzzylogic2000 you fixed test and I updated dependancy and it's actually passing tests and shit :D shall I merge?

**Comment by fuzzylogic2000** [comment]
GO FOR IT!

**Comment by phillimorland** [comment]
Boom!

**Comment by phillimorland** [comment]
That felt good! :)

**Pull request merged**